src/takedown.js

```diff
@@ -0,0 +1,85 @@
+import { getRepliesInBatch, getUserReplies } from './util/graphql.js';
+import { getSpamList } from './getSpamList.js';
+import { getAllPRs, createPullRequest } from './githubPR.js';
+import subTime from 'date-fns/sub';
+
+const knownUsers = new Set();
+
+async function initKnownUsers() {
+  const list = await getAllPRs();
```

Comment:
nit: I think this logic works fine and we can proceed with this solution now. However, I can imagine that as time goes by, the list of PR can be longer and longer.

A more scalable approach may be get the list of Cofacts reply authors first, and use Github REST API [search capability](https://docs.github.com/en/rest/search/search?apiVersion=2022-11-28#search-issues-and-pull-requests) to ensure that the PR for that user does not already exist.